metaseq/hub_utils.py

```diff
@@ -320,13 +320,13 @@ def generate(
                     tokens_no_eos = tokens[1:] if echo else tokens
                     scores_with_eos = [None] + scores[1:] if echo else scores
                     # turn it into a string
-                    text = self.bpe.decode(tokens_no_eos)
+                    text = self.bpe.bpe.decode(tokens_no_eos)
```

Comment:
this wont call the hf_byte_bpe decode, in our case we need to set skip_special_tokens to False to generate speech tokens.